jax_tqdm/pbar.py

```diff
@@ -131,29 +131,29 @@ def _define_tqdm(arg, transform):
         tqdm_bars[0].set_description(message, refresh=False)
 
     def _update_tqdm(arg, transform):
-        tqdm_bars[0].update(arg)
+        tqdm_bars[0].update(int(arg))
 
     def _update_progress_bar(iter_num):
         "Updates tqdm from a JAX scan or loop"
         _ = jax.lax.cond(
             iter_num == 0,
-            lambda _: callback(_define_tqdm, None, None),
+            lambda _: callback(_define_tqdm, None, None, ordered=True),
```

Comment:
Thanks for adding this. Is there a good way to test? Know async issues can be a really pain to test.

Reply:
I'm also not sure of a good way to test, as it's very dependent on the wrapped function, the number of iterations, and the print rate.

Reply:
Yup, i might look to add a very fast function with the highest print-rate. Guess it would also depend on the machines the CI is using. Maybe if run it enough time you have some sort of guarantee it's all working Ok.